packages/app/src/pages/set-password.tsx

```diff
@@ -8,12 +8,19 @@ import { buildEnterKeyEvent } from '@components/link';
 import { useOnboardingState } from '@common/hooks/use-onboarding-state';
 import { USERNAMES_ENABLED } from '@common/constants';
 import { validatePassword, blankPasswordValidation } from '@common/validate-password';
+import { Text } from '@components/typography';
 
 interface SetPasswordProps {
   redirect?: boolean;
   accountGate?: boolean;
+  placeholder?: string;
 }
-export const SetPasswordPage: React.FC<SetPasswordProps> = ({ redirect, accountGate }) => {
+
+export const SetPasswordPage: React.FC<SetPasswordProps> = ({
+  redirect,
+  accountGate,
+  placeholder,
```

Comment:
This never actually gets set by any component. I'd imagine you wanted to try and re-use this on the unlock page? For now I'd remove it.

Reply:
Yes, I added this by accident because i assumed both password views (set one, input one) used the same component. Ideally they would, and as such one would have the placeholder "Set a password" and the other would have "Enter your password" see this edit -> https://github.com/blockstack/ux/pull/1036/files#diff-a84ef9a46341ad44e66466e8404f3f7cb731513dc29c60296cb8d2f898b27fcbR35

Reply:
Yeah that makes sense!

Reply:
Will take care of in a future PR to refactor this